ui/logviewer/helpers.js

```diff
@@ -0,0 +1,99 @@
+// This file is for spitting out something nicely formatted from the Talos logs.
+// This code is taken from https://github.com/gregtatum/scripts/blob/master/mochitest-formatter/from-talos-log.js
+const x256 = require('x256');
+
+function output(line, color) {
```

Comment:
This output function should be returning `HTML` rather than this.  So use the color you get back and style the HTML with it.